packages/babel-helpers/src/helpers.js

```diff
@@ -470,7 +440,7 @@ helpers.construct = () => template.program.ast`
   import setPrototypeOf from "setPrototypeOf";
 
   export default function _construct(Parent, args, Class) {
-    if (typeof Reflect === "object" && Reflect.construct) {
+    if (typeof Reflect !== "undefined" && Reflect.construct) {
```

Comment:
Isn't "object" better? What if Reflect is a number? that will probably never happen tho.

Reply:
`typeof X === "object"` and `=== "symbol"` requires a typeof helper to compensate for polyfilled `Symbol` instances.